examples/components/components-two-way-binding/app/counter.component.ts

```diff
@@ -1,22 +1,17 @@
-import {Component, Input, Output, EventEmitter} from '@angular/core';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
 
 @Component({
-  selector: 'counter',
-  template: `
-    <div>
-
-      <p>
-        <ng-content></ng-content>
-        Count: {{ count }} -
-        <button (click)="increment()">Increment</button>
-      </p>
-
-    </div>
-  `
+  selector: 'rio-counter',
+  templateUrl: 'app/counter.component.html'
 })
-export class Counter {
-  @Input() count: number = 0;
-  @Output() countChange: EventEmitter<number> = new EventEmitter<number>();
```

Comment:
Like on the other PR, is it an Angular style guide practice to initialize in the constructor?